### DISCUSSION TITLE
What benefits does yolov26 offer over yolov11?

### DISCUSSION THREAD

**Discussion by Raj2032** [ultralytics/ultralytics#23329] [Q&A]
I have tested both yolov26 and yolov11 both using small models and accuracy is slightly worse on yolov26 than it is on yolov26 (I did a “proof of concept” with my training).

I was wondering is this the reason due to the trade off favoring slightly higher speeds over accuracy?

I used the exact same paramaters, for example:

`dataset_custom.yaml`

```yaml
train: images/train
val: images/val

nc: 1

names: ["mc"]
```
`train.py`

```py
from ultralytics import YOLO

# model = YOLO("yolo11s.pt")
model = YOLO("yolo26s.pt")
model.train(data = "dataset_custom.yaml", imgsz = 640,
batch = 8, epochs = 100, workers = 0, device = "cpu")
```

I am pretty sure this is optimal for both yolo11s and yolo26s

(I had 11 images for training and 5 images for validation, its just for a proof of concept rather than an official product I am trying to build, hence the lack of image training).

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Raj2032, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

This is an automated response 🤖—an Ultralytics engineer will also assist soon 🙏

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it. In your case, an MRE would ideally include the exact commands/code you ran, the full console logs, the `ultralytics` version, and enough detail for someone to reproduce the YOLO11 vs YOLO26 comparison on the same data/settings.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/). With very small proof-of-concept datasets (like 11 train / 5 val), sharing training curves/logs and a couple of representative sample images can be especially helpful 📌

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Reply by glenn-jocher** [reply]
With only 11 train / 5 val images, the reported mAP/accuracy will have huge variance (and can flip between runs), so a small “YOLO11s > YOLO26s” result is usually noise/overfit rather than an intentional speed-for-accuracy tradeoff; YOLO26’s main gains are simpler end-to-end deployment (NMS-free) and faster CPU inference, as summarized in the [YOLO11 vs YOLO26 FAQ](https://docs.ultralytics.com/help/FAQ/). To make the comparison meaningful, please rerun a few times with a fixed `seed` and share the full training logs plus `yolo checks` output (and if possible a small sample of images/labels), and we can help sanity-check whether you’re just hitting small-data instability or a real regression on your setup.

**Reply by Raj2032** [reply]
Hi mate, when you say to rerun a few times do you mean I should rerun training a few times or rerun the model a few times?

When you say with a fixed seed is this something that should go inside `train.py`? What value should I set it to?

**Reply by glenn-jocher** [reply]
I meant rerun the **training + validation** a few times (not inference), since training is stochastic; set the same `seed` inside `model.train()` for both models (any integer is fine—`0` or `42` are common) and compare the resulting `best.pt` metrics across runs—see the [Train mode arguments](https://docs.ultralytics.com/modes/train/) for details.

```python
from ultralytics import YOLO

model = YOLO("yolo26s.pt")  # or "yolo11s.pt"
model.train(data="dataset_custom.yaml", imgsz=640, batch=8, epochs=100, workers=0, device="cpu", seed=0)
```

**Comment by Y-T-G** [comment]
You should use a larger dataset for more representative results because you are trying to learn millions of parameters from just 11 images.
